cli/cli/kurtosis_gateway/server/api_container_gateway/api_container_gateway_service_server.go

```diff
@@ -56,6 +59,7 @@ func NewEnclaveApiContainerGatewayServer(connectionProvider *connection.GatewayC
 		mutex:                               &sync.Mutex{},
 		userServiceNameToLocalConnectionMap: userServiceToLocalConnectionMap,
 		enclaveId:                           enclaveId,
+		connect:                             kurtosis_core_rpc_api_bindings.Connect_CONNECT,
```

Comment:
nit can we rename this to `userServiceConnect` so that its clear that this is for user services and not the APIC container itself